### PR TITLE
feat: informers and listers for test and test suite

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: pkg/clientset/versioned/fake/clientset_generated.go [hunk 1]
```diff
@@ -77,7 +81,17 @@ var (
 	_ testing.FakeClient  = &Clientset{}
 )
 
-// TestTriggerV1 retrieves the TestsV1Client
+// TestsV1 retrieves the TestsV1Client
```
Comment (vLia) [inline pkg/clientset/versioned/fake/clientset_generated.go:84]: Nice catch on this one 👍

File: pkg/clientset/versioned/typed/tests/v3/fake/fake_tests.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+var testsResource = schema.GroupVersionResource{Group: "tests.testkube.io", Version: "v3", Resource: "Test"}
+
+var testsKind = schema.GroupVersionKind{Group: "tests.testkube.io", Version: "v3", Kind: "Test"}
+
+// List takes label and field selectors, and returns the list of Tests that match those selectors.
+func (c *FakeTests) List(ctx context.Context, opts v1.ListOptions) (result *testsv3.TestList, err error) {
+	obj, err := c.Fake.
+		Invokes(testing.NewListAction(testsResource, testsKind, c.ns, opts), &testsv3.TestList{})
+
+	if obj == nil {
```
Comment (vLia) [inline pkg/clientset/versioned/typed/tests/v3/fake/fake_tests.go:46]: should we also check for err here, or is checking for the empty object enough?

File: pkg/clientset/versioned/typed/tests/v2/fake/fake_testsuite.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	ns   string
+}
+
+var testSuitesResource = schema.GroupVersionResource{Group: "tests.testkube.io", Version: "v2", Resource: "TestSuite"}
+
+var testSuitesKind = schema.GroupVersionKind{Group: "tests.testkube.io", Version: "v2", Kind: "TestSuite"}
+
+// List takes label and field selectors, and returns the list of TestSuites that match those selectors.
+func (c *FakeTestSuites) List(ctx context.Context, opts v1.ListOptions) (result *testsuitev2.TestSuiteList, err error) {
+	obj, err := c.Fake.
```
Comment (vLia) [inline pkg/clientset/versioned/typed/tests/v2/fake/fake_testsuite.go:43]: same here, should the err be checked as well?

File: pkg/clientset/versioned/typed/tests/v3/fake/fake_tests.go (hunk 2 above)
Comment (vsukhin) [reply pkg/clientset/versioned/typed/tests/v3/fake/fake_tests.go:46]: good point, just copied/pasted a lot, need to relook at it carefully ;)

**Pull request merged**